pkg/config/types/types.go

```diff
@@ -547,17 +539,6 @@ func MergeOlvmOck(def *OlvmOck, ovr *OlvmOck) OlvmOck {
 	}
 }
 
-// MergeOlvmControlPlaneEndpoint takes two OlvmControlPlaneEndpoints and merges them into
```

Comment:
Why is this being removed?

Reply:
It wasn't being used.  In fact the struct associated with it (OlvmControlPlaneEndpoint) now has no usages.  I just pushed a change to also remove the struct def.  I will check with @pfmackin to make sure that no longer having this struct was not an oversight when the network configuration changes were made to the OLVMProvider.

Reply:
yes we use the virtualIP this was left over and should be removed.